src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/VariantsToTable.java

```diff
@@ -143,9 +144,19 @@
      * will cause VariantsToTable to write out NA values for missing fields instead of throwing an error.
      * Note that this flag only applies to standard columns (CHROM, ID, QUAL) and the INFO field and it does not apply to the genotype field.
      */
-    @Advanced
-    @Argument(fullName="allowMissingData", shortName="AMD", doc="If provided, we will not require every record to contain every field", optional=true)
+    @Hidden
+    @Deprecated
+    @Argument(fullName="allowMissingData", shortName="AMD", doc="This argument is no longer used, refer to the documentation for --errorIfMissingData", optional=true)
```

Comment:
Although now that I think about it, in GATK4 we could go for a clean slate and just get rid of all the old GATK3 arguments since the upcoming refactoring is going to break all arguments anyway (we're still doing that, right?)